rules/eslint/best-practices/eslint.js

```diff
@@ -1,5 +1,3 @@
-"use strict";
```

Comment:
Added it back, accidentally removed it during my "I wonder if this has anything to do with it" testing.